src/backend/upload_file/upload_file_for_chunks.py

```diff
@@ -0,0 +1,172 @@
+import json
+import os
+import pdfplumber
+import magic
+import easyocr
+import numpy as np
+import logging
+
+#config logging
+logging.basicConfig(level=logging.DEBUG,format='%(asctime)s - %(levelname)s - %(message)s')
+logger = logging.getLogger(__name__)
+
+class CheckFileType:
+
+    def check_file_type(self, filename):
+        mime_to_extension = {
+            "application/pdf": ".pdf"
+        }
+        try:
+            mime = magic.Magic(mime=True)
+            file_type = mime.from_file(filename)
+            extension = mime_to_extension.get(file_type)
+            if not extension:
+                logger.error(f"Unknown MIME type detected: {file_type}")
+                return None
+            return extension
+        except Exception as e:
+            logger.error(f"Error detecting file type for {filename}: {e}")
+            return None
+
+
+class ExtractFiles:
+
+    def __init__(self):
+
+        # Initialize EasyOCR Reader
+        self.ocr_reader = easyocr.Reader(['en'], gpu=False)
+
```

Comment:
add two more functions 

1) Download file from minio bucket to local storage 
download_file(customer_guid, filename)
This function should download the file from minio_bucket to your local storage /tmp/customer_guid/filename

2) Write a function extract_file(customer_guid, filename) and this function will have the below steps
     a) call download_file(customer_guid, filename) written earlier in this comment
     b) identify the filetype of the downloaded file using function def check_file_type(self, filename):
     c) depending on the filetype, we need to call extractors - in this case we have currently written only one extractor - pdf - extract_pdf_with_layout(self, filename, output_json_path):
     d) upload the output extracted file /tmp/customer_guid/example.pdf.rawcontent back to minio_bucket customer_guid